src/components/App/App.jsx

```diff
@@ -2,34 +2,25 @@ import React from 'react'
 import { Provider } from 'react-redux'
 import PropTypes from 'prop-types'
 
-import { CozyProvider } from 'cozy-client'
-import SharingProvider from 'cozy-sharing'
-import { I18n } from 'cozy-ui/transpiled/react/I18n'
-import { BreakpointsProvider } from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
-
+import DriveProvider from 'drive/lib/DriveProvider'
 import { ThumbnailSizeContextProvider } from 'drive/lib/ThumbnailSizeContext'
 import { ModalContextProvider } from 'drive/lib/ModalContext'
 import { AcceptingSharingProvider } from 'drive/lib/AcceptingSharingContext'
-import StyledApp from 'drive/web/modules/drive/StyledApp'
 
 const App = props => {
   return (
     <Provider store={props.store}>
-      <I18n lang={props.lang} polyglot={props.polyglot}>
-        <CozyProvider client={props.client}>
-          <SharingProvider doctype="io.cozy.files" documentType="Files">
-            <AcceptingSharingProvider>
-              <ThumbnailSizeContextProvider>
-                <ModalContextProvider>
-                  <BreakpointsProvider>
-                    <StyledApp>{props.children}</StyledApp>
-                  </BreakpointsProvider>
-                </ModalContextProvider>
-              </ThumbnailSizeContextProvider>
-            </AcceptingSharingProvider>
-          </SharingProvider>
-        </CozyProvider>
-      </I18n>
+      <DriveProvider
+        client={props.client}
+        lang={props.lang}
+        polyglot={props.polyglot}
+      >
+        <AcceptingSharingProvider>
+          <ThumbnailSizeContextProvider>
+            <ModalContextProvider>{props.children}</ModalContextProvider>
+          </ThumbnailSizeContextProvider>
+        </AcceptingSharingProvider>
+      </DriveProvider>
     </Provider>
   )
 }
```

Comment:
J'ai laissé quelques providers ici, si nous en n'avons pas besoin dans l'intent, inutile de les rajouter, non ?

Reply:
Pas besoin du ProviderRedux sur l'intent ? Ça fonctionne bien as is?

Reply:
Tu l'as mis sur App.jsx c'est aussi mis pour les autres targets ? genre public ?

Reply:
- A priori pas besoin du ProviderRedux sur l'intent, je n'ai rien enlevé dans mes changements
- Pour le public c'est assez spécifique, mais on fait appel à App (qui utilise DriveProvider), j'ai supprimé les 2 doublons du coup (mea culpa j'avais regardé un peu trop vite)